src/main/java/dev/bernasss12/bebooks/client/gui/ModConfig.java

```diff
@@ -330,20 +337,32 @@ public StoredEnchantmentData(EnchantmentData data) {
     }
 
     public static class EnchantmentData extends StoredEnchantmentData {
-        @NotNull public String translatedName;
         public Enchantment enchantment;
+        private String translatedName;
 
-        public EnchantmentData(Enchantment enchantment, @NotNull String translatedName, int index, int color) {
+        public EnchantmentData(Enchantment enchantment, int index, int color) {
             super(index, color);
             this.enchantment = enchantment;
-            this.translatedName = translatedName;
         }
 
         public EnchantmentData(StoredEnchantmentData data, String key) {
             super(data.orderIndex, data.color);
-            Optional<Enchantment> enchantment = Registry.ENCHANTMENT.getOrEmpty(Identifier.tryParse(key));
-            enchantment.ifPresent(value -> this.enchantment = value);
-            this.translatedName = enchantment.isPresent() ? enchantment.get().getName(1).getString() : "Error translating.";
+            this.enchantment = Registry.ENCHANTMENT.get(Identifier.tryParse(key));
```

Comment:
I never even thought of this being possible always worked around the assumption that if an enchantment is on an item then it must exist and be registered, nicely spotted!

Reply:
Yeah, I only ran into it because MC Dungeons Weapons had an item with a typoed enchantment on it.
I'm fairly sure enchantments are stored/loaded regardless of the registry, so disabling some mods could leave you with such items too.

Reply:
Yeah that could also happen indeed, I never really tested the mod with other mods that add enchants although in hindsight I probably should have